doozer/doozerlib/rhcos.py

```diff
@@ -13,85 +13,9 @@
 from doozerlib.repodata import OutdatedRPMFinder, Repodata
 from doozerlib.runtime import Runtime
 from doozerlib.util import brew_suffix_for_arch, isolate_el_version_in_release
+from artcommonlib import rhcos
 
 RHCOS_BASE_URL = "https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/storage/releases"
```

Comment:
This constant is also defined in `elliottlib/constants.py` and could be moved in a common place (`artcommonlib/constants.py`?)

Reply:
sure... a few others as well